Resources/Prototypes/Catalog/Fills/Boxes/security.yml

```diff
@@ -70,166 +70,4 @@
         amount: 10
   - type: Sprite
     layers:
-      - state: box_security
-
-# TODO: THESE ARE BAD AND ARE DEPRECATED, DON'T USE THEM PLEASE
```

Comment:
These should not be using storagefill, hence the comment.

Reply:
What do we use to fill a container other than storage fill? Otherwise are we just removing boxes of shells?

Reply:
It should just use an ammo provider. The issue is armouries often have many boxes and it'll probably have pop-in just due to having a few hundred unnecessary entities in one spot.

Reply:
is this something peptide can actually deal with in this PR

Reply:
Add on ballisticammoprovider, set spawn prototype and count? Don't foresee any dramas from it.